From 2b8da8771e70f1767dd79c5a0c71b695f6a172fb Mon Sep 17 00:00:00 2001
From: Ferry Toth <ftoth@exalondelft.nl>
Date: Fri, 18 Dec 2020 21:11:58 +0100
Subject: [PATCH] site: Make sys_siglist default to no

glibc 2.32 has deprecated sys_siglist preventing build on Ubuntu Groovy (20.10)
Suggested-by: Willem-Jan de Hoog <wdehoog@exalondelft.nl>

Signed-off-by: Ferry Toth <ftoth@exalondelft.nl>
---
 meta/site/common-glibc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meta/site/common-glibc b/meta/site/common-glibc
index 1443f8e528..b45178c261 100644
--- a/meta/site/common-glibc
+++ b/meta/site/common-glibc
@@ -1,5 +1,5 @@
 # general
-ac_cv_have_decl_sys_siglist=${ac_cv_have_decl_sys_siglist=yes}
+# ac_cv_have_decl_sys_siglist=${ac_cv_have_decl_sys_siglist=yes}
 ac_cv_func_realloc_works=${ac_cv_func_realloc_works=yes}
 ac_cv_func_realloc_0_nonnull=${ac_cv_func_realloc_0_nonnull=yes}
 ac_cv_func_malloc_works=${ac_cv_func_malloc_works=yes}
-- 
2.27.0

